firmware/controllers/can/obd2.cpp

```diff
@@ -156,11 +156,9 @@ static void handleGetDataRequest(const CANRxFrame& rx, size_t busIndex) {
 		obdSendValue(_1_MODE, pid, 1, Sensor::getOrZero(SensorType::Tps1) * ODB_TPS_BYTE_PERCENT, busIndex);	// (A*100/255)
 		break;
 	case PID_FUEL_AIR_RATIO_1: {
-		float lambda = Sensor::getOrZero(SensorType::Lambda1);
-		// phi = 1 / lambda
-		float phi = clampF(0, 1 / lambda, 1.99f);
+		float lambda = clampF(0, Sensor::getOrZero(SensorType::Lambda1), 1.99f);
```

Comment:
Just curious, why `1.99f` vs e.g. `1.9999f` -- I realize "< 2 ratio" is the concern; is the extra "top end" not really worth?

Reply:
Nope. Not much reason to unless you're doing "real" diagnosis using OBD2. We support OBD2 pretty much exclusively for gauges/dashes/apps/etc, not real diagnosis.

Reply:
> We support OBD2 pretty much exclusively for gauges/dashes/apps/etc, not real diagnosis.

This statement is an important fact but it seems not captured in doc yet.
Is it ok that I add it to https://wiki.rusefi.com/CAN/ ?